addons/web_editor/static/tests/banner_tests.js

```diff
@@ -123,7 +123,8 @@ QUnit.module(
                 `<p>Test</p><div class="o_editor_banner o_not_editable lh-1 d-flex align-items-center alert alert-info pb-0 pt-3" role="status" data-oe-protected="true" contenteditable="false">
                         <i class="fs-4 fa fa-info-circle mb-3" aria-label="Banner Info"></i>
                         <div class="w-100 ms-3" data-oe-protected="false" contenteditable="true">
-                            <p placeholder=\"Type &quot;/&quot; for commands\" class=\"oe-hint oe-command-temporary-hint\"><br></p></div></div><p><br></p>`,
+                            <p placeholder=\"Type &quot;/&quot; for commands\" class=\"oe-hint oe-command-temporary-hint\"><br></p></div>
+                    </div><p><br></p>`,
```

Comment:
Runbot breaks without this change.
I asked Dhaval reason of not adding `unformat` initially when he created this he said he wanted to have `\n          ` invisible nodes.

Reply:
It's kind of weird, isn't it? It would mean there is an invisible `\n               ` that is created or not deleted anymore due to the changes in the PR.

Reply:
Or perhaps it has been resolved now that it matches the two test cases above.

Reply:
I checked previously without my fix when doing a ctrl+A within banner the focusNode used to be `div.w-100.ms-3` and after `range.extractContent` in deleteRange the **end used to be the invisible node after `div.w-100`** the invisible nodes at end are removed in [deleteRange](https://github.com/odoo/odoo/blob/17.0/addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js#L2223C1-L2227C10) but now after my fix the focusNode is the last `<p>` tag within the `div.w-100.ms-3` so here after `range.extractContent` the **end would be the last node (invisible text node) within the `div.w-100.ms-3`**.